httplib_test.go

```diff
@@ -37,15 +36,22 @@ func TestReplyJSON(t *testing.T) {
 			"}"
 	)
 
-	testCase := func(t *testing.T, err error) {
-		recorder := httptest.NewRecorder()
-		replyJSON(recorder, errCode, err)
-		require.Equal(t, expectedErrorResponse, recorder.Body.String())
+	for _, tc := range []struct {
+		desc string
+		err  error
+	}{
+		{"plain error", errors.New("test error")},
+		{"trace error", &TraceErr{Err: errors.New("test error")}},
+		{"trace error with stacktrace", &TraceErr{Err: errors.New("test error"), Traces: Traces{{Path: "A", Func: "B", Line: 1}}}},
+	} {
+		tc := tc
+		t.Run(tc.desc, func(t *testing.T) {
+			t.Parallel()
```

Comment:
I would keep the `t.Parallel()` up above. These subtests are going to complete so fast there's little reason to parallelize them.